### ISSUE TITLE
Default opening mode

### ISSUE THREAD

**Issue by clbarnes** [constantinpape/z5#166]
In h5py v2.10 (2019-09-06), opening without an explicit mode raises a deprecation warning (which can be silenced with an environment variable or global config setting). From h5py 3.0, the default mode will be readonly. https://docs.h5py.org/en/latest/whatsnew/2.10.html#deprecations

z5py may want to do the same for compability / because readonly is the sensible default anyway.

**Comment by constantinpape** [comment]
Good point. I think this is something to consider for the next minor release.

**Comment by constantinpape** [comment]
This will be addressed in the next release.
